tools/pages/permit-holders/types.ts

```diff
@@ -109,3 +110,11 @@ export type GetPermitHolderResponse = {
     >;
   };
 };
+
+/**
+ * Type for data required in Set Permit Holder Status modal
+ */
+export type SetPermitHolderStatusModalData = {
+  readonly id: number;
```

Comment:
`id` is currently not being used in the `SetPermitHolderStatusModal` but will (probably) be required in the API hookup so I included it in this type